### PR TITLE
Created two_sum() function.

### PR DESCRIPTION
**Pull request by arielwilliams**
I made the two_sum() function!

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,7 @@
-def twoSum(n, t):
-    pass
+def two_sum(n, t):
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```
Comment (purplem00n) [inline functions/two_sum.py:3]: can you do better than a nested loop for time complexity?

File: functions/two_sum.py [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+            if n[j] == t - n[i]:
+                return [i, j]
+
+
```
Comment (purplem00n) [inline functions/two_sum.py:7]: would love to see what tests you made!

**Review by purplem00n — CHANGES_REQUESTED** [review]

File: functions/two_sum.py (hunk 1 above)
Comment (arielwilliams) [reply functions/two_sum.py:3]: Yes, I thought about this but it's too much work for me right now...

File: functions/two_sum.py (hunk 2 above)
Comment (arielwilliams) [reply functions/two_sum.py:7]: Yes, I did forget those unfortunately. I will not be making a new code review for the updates though. Time constraints...

**Pull request merged**